sdk/src/main/java/com/mapbox/maps/MapboxConstants.kt

```diff
@@ -7,7 +7,7 @@ import java.util.*
 /**
  * Name of the database file.
  */
-const val DATABASE_NAME = "mbx.db"
+const val DATABASE_NAME = "mapbox/maps/ambient_cache.db"
```

Comment:
Please verify the mapbox/maps folder is created, we need to create the path before passing to gl-native. The limitation is ticketed upstream.